CMakeLists.txt

```diff
@@ -532,9 +532,6 @@ if(AVIF_BUILD_TESTS)
     add_executable(aviftest
         apps/shared/y4m.c
         tests/aviftest.c
-        tests/cJSON.c
```

Comment:
Please remove the cJSON license from the LICENSE file.

Reply:
Done!